Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
Maybe it's better to use `python3-5` image than try to patch all the missing dependencies ? :P